source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
+
+Archlinux uses a specific feature called [Arch User Repository](https://wiki.archlinux.org/index.php/Arch_User_Repository) to manage extra packages which do not belong to the official repository. It allows you to compile a package from source and then install it via the Arch package manager [pacman](https://wiki.archlinux.org/index.php/Pacman#Additional_commands). This way you will be able to track easily upgrades or dependency issues. This is the recommended way to install extra packages.
```

Comment:
Well, again, this "unofficial" story! Using AUR is the Arch recommended way to install extra package. This practice is encouraged more than building from source the traditional way (git clone, make & make install). AUR allows these extra packages to be managed by the ARCH package manager, which is very beneficial (version following, dependencies).
If you want, I may stress using AUR is the recommended way for Archlinux and not for mattermost.